CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 4.3.3 (2020-09-11)
+
+- Fixed the tabset component to allow activating a tab as soon as it is created. [#132](https://github.com/blackbaud/skyux-tabs/pull/132)
```

Comment:
How about "Fixed the tabset component to activate tabs as soon as they're created."  ?

Reply:
That changes the meaning a bit. The problem we're fixing is _allowing_ consumers to activate a new tab when it is created. We're not activating tabs when they're created; the problem only comes when a consumer wishes to activate a new tab immediately after being created.

Reply:
Gotcha. Can we include that then? 

Fixed the tabset component to allow consumers to activate a tab as soon as they create it.

Reply:
Done.